src/data/media-player.ts

```diff
@@ -147,6 +148,7 @@ export const MediaClassBrowserSettings: {
 
 export interface MediaPickedEvent {
   item: MediaPlayerItem;
+  navigateIds?: MediaPlayerItemId[];
```

Comment:
should we make this mandatory and instead have the TTS form use a different event?